lindeck-frontend/src/views/boody/deckPage/editDeckDialog/DeckEditDialog.jsx

```diff
@@ -0,0 +1,94 @@
+import React, {useRef} from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import "./DeckEditDialog.css"
+import TextField from "@material-ui/core/TextField";
+
+export default function DeckEditDialog(props) {
```

Comment:
Also logic for deck editing will be much more complicated, so i will review it more attentively lately.

Reply:
kk